docs/Catalog/VisualizationDashboards.md

```diff
@@ -67,6 +67,7 @@ or an object with one or more of the following properties:
 - `path` - file path relative to `quilt_summarize.json`
 - `title` - title rendered instead of file path
 - `description` - description in markdown format
+- `expanded` - expanded file view state (default is `false`)
```

Comment:
I know this is a code change but I think expand is clearer (nice-to-have).
```suggestion
- `expand` - Display the file (`true`) or display a preview in an expandable box (`false`, default)
```

Reply:
https://github.com/quiltdata/quilt/pull/2783